### PR TITLE
Cleanup whitespace in ssl_lib.c (tabs to spaces)

### PR DESCRIPTION
**Pull request by mspncp**
Nothing much to be said about this commit.

### CODE REVIEW

File: ssl/ssl_lib.c [hunk 1]
```diff
@@ -454,44 +454,44 @@ static int ssl_check_allowed_versions(int min_version, int max_version)
             return 0;
     } else {
         /* Regular TLS version checks. */
-	if (min_version == 0)
-	    min_version = SSL3_VERSION;
-	if (max_version == 0)
-	    max_version = TLS1_3_VERSION;
+    if (min_version == 0)
```
Comment (kaduk) [inline ssl/ssl_lib.c]: Starting here I think we need another 4 spaces of indentation.

File: ssl/ssl_lib.c [hunk 2]
```diff
@@ -454,44 +454,44 @@ static int ssl_check_allowed_versions(int min_version, int max_version)
...
-	    min_version = TLS1_3_VERSION;
+    if (min_version == TLS1_2_VERSION)
+        min_version = TLS1_3_VERSION;
 #endif
-	/* Done massaging versions; do the check. */
-	if (0
+    /* Done massaging versions; do the check. */
+    if (0
 #ifdef OPENSSL_NO_SSL3
             || (min_version <= SSL3_VERSION && SSL3_VERSION <= max_version)
```
Comment (kaduk) [inline ssl/ssl_lib.c]: I think these lines also should get deindented.

File: ssl/ssl_lib.c (hunk 1 above)
Comment (mspncp) [reply ssl/ssl_lib.c]: Ok, I'll have a closer look at the code. Originally, I was working on something else and just did a 'whitespace-cleanup' in Emacs, when I noticed these extra changes. So to keep them out of my PR, I created a new one.

File: ssl/ssl_lib.c (hunk 2 above)
Comment (mspncp) [reply ssl/ssl_lib.c]: No, the "if (0" needs to be indented. (You'll see in a minute when I commit the changes)

**Comment by mspncp** [comment]
Looks better to me. I hope everything is correct now.

**Comment by kaduk** [comment]
@dot-asm can you merge?  I am having trouble passing the commit hook again with this one.

**Comment by dot-asm** [comment]
> I am having trouble passing the commit hook again with this one.

So am I... Hmm... It's midnight here, so I better postpone this till morning....

**Comment by dot-asm** [comment]
Committed. Thanks.

[Reason for failure was mixed-case e-mail address in our CLA list.]

**Comment by mspncp** [comment]
@dot-asm it looks like there is still some case sensitivity issue with my email address. Did you change it to lowercase in the cla list ? Because now the cla-check fails again, see PR #4402.

**Comment by dot-asm** [comment]
> Did you change it to lowercase in the cla list?

Yes, otherwise it was impossible to merge *this* request. But take it easy, it will be sorted out...

**Comment by mspncp** [comment]
Don't worry, I am accustomed to computers having problems with my last name ;-)

**Pull request closed without merging**